packages/js/date/src/test/index.ts

```diff
@@ -71,19 +79,25 @@ describe( 'toMoment', () => {
 
 	it( 'should handle isoFormat dates', () => {
 		const myMoment = toMoment( 'YYYY', '2018-04-15' );
+		if ( myMoment === null ) fail( 'it should not reach here' );
```

Comment:
Nitpick but I'd prefer the fail message to be more precise, i.e: "myMoment should not be null" or something like that

Reply:
@ilyasfoo Thanks for the review. 🙌  Changed them in 6c4674fe9ab8445a07a1d609f6d8308fcc0bd86a